src/main/java/it/pagopa/pn/commons/utils/ClientAspectLogging.java

```diff
@@ -0,0 +1,89 @@
+package it.pagopa.pn.commons.utils;
+
+import lombok.CustomLog;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+@Aspect
+@CustomLog
+public class ClientAspectLogging {
+
+    private static final String SENSITIVE_DATA = "<Hidden Data>";
+
+    @Pointcut("execution(* it.pagopa.pn..generated.openapi.msclient..api.*.*(..))")
+    public void client() {
+        // all client methods
+    }
+
+    @Around(value = "client()")
+    public Object logAroundClient(ProceedingJoinPoint joinPoint) throws Throwable {
+        Object[] arguments = Arrays.stream(joinPoint.getArgs()).toArray();
```

Comment:
Già joinPoint.getArgs() dovrebbe essere di tipo Object[]. Possiamo evitare una operazione in più